azure-cli/plan.sh

```diff
@@ -26,32 +26,33 @@ pkg_deps=(
 pkg_bin_dirs=(bin)
 
 do_prepare() {
-  python -m venv "$pkg_prefix"
-  source "$pkg_prefix/bin/activate"
+  python -m venv "${pkg_prefix}"
+  source "${pkg_prefix}/bin/activate"
 
   pip install --upgrade --force-reinstall pip
   pip install wheel
 }
 
 do_build() {
-  for d in src/azure-cli src/azure-cli-telemetry src/azure-cli-core src/azure-cli-nspkg src/azure-cli-command_modules-nspkg src/command_modules/azure-cli-*/ ; do
+  attach
```

Comment:
Hi @predominant.  Could you try removing this 'attach' and committing.  I think this is why the buildkite is failing.

Reply:
Done!

Reply:
Tests all pass for me again.  Thanks.